sql/src/main/java/io/cloudevents/sql/impl/ParserImpl.java

```diff
@@ -27,7 +31,9 @@ public static Parser getInstance() {
         return ParserImpl.SingletonContainer.INSTANCE;
     }
 
-    public ParserImpl() {
+    // TODO this is super bad, remove it!
+    public ParserImpl(boolean constantFolding) {
```

Comment:
I agree! :)

Reply:
yeah I wanted to create some builder like thing

Reply:
Can you check it out now? I've added a ParserBuilder